website/sandbox.config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "infiniteLoopProtection": true,
+  "hardReloadOnChange": true,
+  "view": "browser",
+  "template": "docusaurus",
+  "node": "16",
```

Comment:
we should probably also set the `engines` field in `package.json`

Reply:
https://github.com/valence-rs/valence/blob/537ebbb71ddf4451961e136aaa98c9985cd4ea26/website/package.json#L42
it is set already, if this is what you mean